mappings/net/minecraft/world/WorldEvents.mapping

```diff
@@ -128,6 +129,7 @@ CLASS net/minecraft/unmapped/C_kkbhcwdu net/minecraft/world/WorldEvents
 		COMMENT A Wet Sponge dries out in a hot dimension.
 		COMMENT <br>Spawns cloud particles.
 		COMMENT <p>Called by {@link net.minecraft.block.WetSpongeBlock#onBlockAdded(net.minecraft.block.BlockState, net.minecraft.world.World, net.minecraft.util.math.BlockPos, net.minecraft.block.BlockState, boolean) WetSpongeBlock#onBlockAdded}
+	FIELD f_kmmyanss SNIFFER_EGG_CRACKS I
```

Comment:
Hm some docs on this and the other new events would be nice

Reply:
Yeaaaaaaahhhhh I was afraid you'd say that

Reply:
doesn't have to come in this pr, but yeah that should be on the todo list for sure